tests/test_peregrine_cli.py

```diff
@@ -20,6 +20,7 @@ def test_peregrine_cli():
     myna_path = os.path.abspath(os.path.join(test_path, ".."))
     build_dir = os.path.join(myna_path, "examples")
     output_dir = os.path.join(build_dir, "Myna")
+    temp_dir = os.path.abspath("myna_tmp")
```

Comment:
I always see temp = temperature, but that's just me

Reply:
Fixed, at least to be consistent within the same file.